### PR TITLE
Dashboard landing 1

### PR DESCRIPTION
**Pull request by rachel-ftw**

### CODE REVIEW

**Comment by rachel-ftw** [comment]
WIP, dashboard page

File: src/components/Route.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import React, { Component } from 'react'
+import { Router, Route, IndexRoute, hashHistory } from 'react-router'
+import ReactDOM from 'react-dom'
+
+ReactDOM.render(
+  <Router history={hasHistory}>
```
Comment (mantinone) [inline src/components/Route.js]: Typo, should be hashHistory

File: src/components/Dashboard.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import React, { Component } from 'react'
+import DashNav from './DashNav'
+import LocalNav from './LocalNav'
+import DashBody from './DashBody'
+
+class Dashboard extends Component {
+  render() {
+    return (
+      <div className="dashboard-home">
+        <DashNav/>
```
Comment (mantinone) [inline src/components/Dashboard.js]: Things to rename: 

- DashNav -> DashboardNav
- DashBody -> DashboardBody
- DashWidget -> DashboardWelcome

File: src/components/DashNav.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+
+class DashNav extends Component {
+  render () {
+    return (
+      <nav className="dash-nav">
+        <div className="dashboard-menu"> &#9776; Dashboards </div>
+        <div className="dashboard-right">
+          <div className="logo-white-square">
+            <div>D</div>
```
Comment (mantinone) [inline src/components/DashNav.js]: Render this D logo on the far right

File: src/styles/button.css [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+  text-decoration: none;
+  color: #fff;
+  text-align: center;
+  border-radius: 3px;
+  padding: 1em 2em;
+  box-shadow: none;
+  display: inline-block;
+}
+
+@media only screen and (max-width: 720px) {
```
Comment (mantinone) [inline src/styles/button.css]: Why are we doing media queries and so many breakpoints? Geckoboard is very much not responsive . . . also, are these doing anything?

File: src/styles/dash_nav.css [hunk 5]
```diff
@@ -0,0 +1,122 @@
...
+  margin-right: 10px;
+  opacity:0;
+}
+
+.account-icon {
+  background-color: #F0F2F3;
+  height: 33px;
+  width: 33px;
+  min-width: 33px;
+  border-radius: 2px;
```
Comment (mantinone) [inline src/styles/dash_nav.css]: Should be `border-radius: 50%`

File: src/components/LocalNav.js [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+    return(
+      <nav className="local-nav">
+        <div className="local-nav-left"> Company Dasboard </div>
+        <div className="local-nav-right"></div>
+          <img className="logo-type" width="30" src="data:image/svg+xml;base64,PD94bWwgdmVyc2lvbj0iMS4wIiBlbmNvZGluZz0idXRmLTgiPz4NCjwhLS0gR2VuZXJhdG9yOiBBZG9iZSBJbGx1c3RyYXRvciAxNi4wLjAsIFNWRyBFeHBvcnQgUGx1Zy1JbiAuIFNWRyBWZXJzaW9uOiA2LjAwIEJ1aWxkIDApICAtLT4NCjwhRE9DVFlQRSBzdmcgUFVCTElDICItLy9XM0MvL0RURCBTVkcgMS4xLy9FTiIgImh0dHA6Ly93d3cudzMub3JnL0dyYXBoaWNzL1NWRy8xLjEvRFREL3N2ZzExLmR0ZCI+DQo8c3ZnIHZlcnNpb249IjEuMSIgaWQ9IkxheWVyXzEiIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyIgeG1sbnM6eGxpbms9Imh0dHA6Ly93d3cudzMub3JnLzE5OTkveGxpbmsiIHg9IjBweCIgeT0iMHB4Ig0KCSB3aWR0aD0iNTEycHgiIGhlaWdodD0iNTEycHgiIHZpZXdCb3g9IjAgMCA1MTIgNTEyIiBlbmFibGUtYmFja2dyb3VuZD0ibmV3IDAgMCA1MTIgNTEyIiB4bWw6c3BhY2U9InByZXNlcnZlIj4NCjxwYXRoIGZpbGw9IiNmZmZmZmYiIGQ9Ik0yNTYuMDMxLDI4OHYtNjMuNTYzYy02OC44MTMsMi4xMjUtMTI0LDU4LjIxOS0xMjQsMTI3LjU2M3M1NS4xODgsMTI1LjQwNiwxMjQsMTI3LjU5NFY0ODANCgljLTEwNi4wNjMsMC4wMzEtMTkyLjA2My04NS45MzgtMTkyLjA2My0xOTJzODYtMTkyLjAzMSwxOTIuMDYzLTE5MlYzMmwxOTIsMTI4TDI1Ni4wMzEsMjg4eiIvPg0KPC9zdmc+DQo=" role="presentation"/>
```
Comment (mantinone) [inline src/components/LocalNav.js]: Why would we do this thing??

File: src/components/LocalNav.js [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+          <a href="#" className="share">Share</a>
```
Comment (mantinone) [inline src/components/LocalNav.js]: Anchor tags will eventually need to become `<Link>`s. But we can save that for a future PR

File: src/components/DashBody.js [hunk 8]
```diff
@@ -0,0 +1,18 @@
+import React, { Component } from 'react'
+import DashWidget from './DashWidget'
+import '../styles/dash_body.css'
+
+class DashBody extends Component {
+  render() {
+    return (
+      <div className="dash-main">
+        <DashWidget/>
+        <div className="chat-icon">
```
Comment (mantinone) [inline src/components/DashBody.js]: Another inline image to replace. Also, let's get the chat icon into its own component.

**Pull request closed without merging**